src/layout/mod.rs

```diff
@@ -1952,6 +1953,13 @@ impl<W: LayoutElement> Layout<W> {
         monitor.expel_from_column();
     }
 
+    pub fn swap_window_in_direction(&mut self, direction: ScrollDirection) {
```

Comment:
Please add this to the `Op` test enum below so it can participate in randomized tests.